tests/Integration/Resources/Storage/Test_Resources_GFAL2StorageBase.py

```diff
@@ -42,11 +45,11 @@
 DESTINATION_PATH = ''
 
 try:
-  res = getProxyInfo()
-  if not res['OK']:
-    gLogger.error( "Failed to get client proxy information.", res['Message'] )
+  proxyInfoRes = getProxyInfo()
```

Comment:
Because of debugging: this test does not work in Jenkins and I want to see more.